### ISSUE TITLE
Loader falls back to default config if here is not a file to load

### ISSUE THREAD

**Issue by LexiconCode** [dictation-toolbox/natlink#126]
Loader falls back to default config if not a a file (_a-grammer.py) to load despite a user dir path being set in a existing config.

Reproduce
1. Use latest master
2. Use natlinkconfig_gui.py to set dragonfly user directory
3. Check in the dragonfly user directory making sure it does not have any file prefixed with '_' for the loader to load.
4. Start Dragon and see issue
![image](https://user-images.githubusercontent.com/24551569/186465426-00c7dbbf-5268-4d87-aa24-5da9d0379db2.png)
- Note: To test use a blank file '_test.py' in the dragonfly user directory. Then the loader behaves as expected using existing config.

**Comment by quintijn** [comment]
Yes, this is not right. But if you drop one valid grammar file in that directory, it will work. (eg from the (path/to/clone/of/natlinkcore)\src\natlinkcore\SampleMacros\OriginalSampleMacros)

I will think about a fix...
